OSVectorTileAPI/StylingVTS/ExternalFile/map.js

```diff
@@ -0,0 +1,107 @@
+//
+// Setup the EPSG:27700 (British National Grid) projection
+//
+proj4.defs("EPSG:27700", "+proj=tmerc +lat_0=49 +lon_0=-2 +k=0.999601 +x_0=400000 +y_0=-100000 +ellps=airy +towgs84=446.448,-125.157,542.060,0.1502,0.2470,0.8421,-20.4894 +datum=OSGB36 +units=m +no_defs");
+ol.proj.proj4.register(proj4);
+var bng = ol.proj.get('EPSG:27700');
+bng.setExtent([-238375.0,0,700000,1300000]);
+
+var map;
+var serviceUrl = "https://osdatahubapi.os.uk/OSVectorTileAPI/vts/v1";
+
+function setupLayer() {
+    if(map) {
+        map.setTarget(null);
+    }
+
+    var key = document.getElementById('keyInput').value;
+    var message = document.getElementById('message');
+    var instructions = document.getElementById('instructions');
+
+    if(!key) {
+        message.classList.add("warning");
+        message.textContent = 'To view the map, please enter a valid API key.';
+        instructions.classList.remove("hidden");
+        return;
+    }
+    message.classList.remove("warning");
+    message.textContent = 'To view the map, please enter a valid API key.';
+    instructions.classList.add("hidden");
+
+    var promise1 = fetch(serviceUrl + '?key=' + key).then(response => response.json());
+    var promise2 = fetch(serviceUrl + '/resources/styles?key=' + key).then(response => response.json());
+    var promise3 = fetch(serviceUrl + '/resources/sprites/sprite.json?key=' + key).then(response => response.json());
+    var spriteImageUrl = serviceUrl + '/resources/sprites/sprite.png?key=' + key
+
+    Promise.all([promise1, promise2, promise3])
+        .then(results => {
+            var service = results[0];
+//            var style = results[1];
```

Comment:
Given we don't use the style we loaded, we shouldn't bother making the http request (line 32). The same comment applies to the InCode example

Reply:
Given my later comment (below), we probably should be loading the json file using that promise on line 32... but we should remove the promise from the InCode example

Reply:
I was keeping it as a trade off to make minimal code changes for the styling to be functional. That said removing promise2 doesn't make a difference to the user and makes it clear that the previous promise2 is not needed in this example.
Done - except for JSON loading. In next commit.